src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY_PREFIX = "buy,";
+    private static final String SUPPLY_PREFIX = "supply,";
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+    private static final int INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File readFile = new File(fromFileName);
+        File writeFile = new File(toFileName);
+
+        calculateReport(readFile, writeFile);
```

Comment:
It's better to have. Each methods should have their own separate logic and be independent
String[] fileInfoArr = readFromFile(readFile); 
String report = getResult(fileInfoArr);
writeToFile(report, toFileName);